content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst

```diff
@@ -0,0 +1,80 @@
+activities such as email or SMS campaigns. To get started, navigate to the :guilabel:`Workflow`
+area, and click :guilabel:`Add New Activity`. A blank activity template will appear where
+parameters can be set for that specific activity.
+
+When :guilabel:`Add New Activity` is selected, a blank activity template will appear.
+
+.. image:: workflow_activities/activity-template.png
+   :align: center
+   :alt: An activity template in Odoo Marketing Automation.
+
+- :guilabel:`Activity Name`: the title of the activity.
+- :guilabel:`Activity Type`: choose between Email, Server Action (internal Odoo operation), or SMS.
+- :guilabel:`Mail Template`: choose from pre-configured templates (or create a new one on-the-fly).
+- :guilabel:`Trigger`: choose when this activity should be triggered.
+- :guilabel:`Expiry Duration`: configure to stop the actions after a specific amount of time
+  (after the scheduled date). When selected, a :guilabel:`Cancel after` field appears, in which the
+  user can choose how many :guilabel:`Hours, Days, Weeks, or Months` they want the actions to cease
+  after the initial date.
+- :guilabel:`Activity Filter`: domain related to this activity (and all subsequent Child Activities).
+- :guilabel:`Applied Filter`: activity will *only* be performed if it satisfies the specified
+  domain (filter).
+
+After the activity's settings are fully configured, click :guilabel:`Save & Close` to finish
+creating the activity.
+
+Workflow activity
+=================
+
+Once saved, the :guilabel:`Workflow Activity` will appear in Odoo. Each activity will be shown as a
+graph (by default). The configured trigger time for that activity can be found to the left of the
+:guilabel:`Workflow Activity` card.
+
+In the :guilabel:`Workflow Activity` section, there is also the number of activities that are
+successful (and rejected). The :guilabel:`Success` (green) and :guilabel:`Rejected` (red) numbers
+are color-coded in the graph for easy reference. 
+
+.. image:: workflow_activities/workflow-activity.png
+   :align: center
+   :alt: Typical workflow activity in Odoo Marketing Automation.
+
+Beneath the :guilabel:`Workflow Activity` data, there are more in-depth analytics detailing how
+many emails (if the :guilabel:`Activity Type` of the activity is set to :guilabel:`Email`) have 
+been :guilabel:`Sent`, and what percentage of those have been :guilabel:`Clicked`, 
+:guilabel:`Replied` to, or :guilabel:`Bounced`.
+
+Child activities
+================
+
+Odoo also has the option to :guilabel:`Add Child Activity`. 
+
+*Child Activities* are sub-activities that are connected to (and triggered by) the 
+activity above it, which is also known as its *Parent Activity*.
+
+Odoo provides a number of triggering options to launch a :guilabel:`Child Activity` - all of which
+depend on the trigger configurations related to the parent activity. 
+
+Under the desired parent activity, hover over :guilabel:`Add child activity`, and select the
+trigger:
```

Comment:
Suggestion: group like-content together, as some of these sentences don't stand well on their own without surrounding context. Source: [content guidelines re: paragraphs](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#paragraphing)

```suggestion
Odoo also has the option to :guilabel:`Add Child Activity`. *Child Activities* are sub-activities
that are connected to (and triggered by) the activity above it, which is also known as its *Parent
Activity*.

Odoo provides a number of triggering options to launch a :guilabel:`Child Activity` - all of which
depend on the trigger configurations related to the parent activity. Under the desired parent
activity, hover over :guilabel:`Add child activity`, and select the trigger:
```